sql/moz-fx-data-shared-prod/telemetry_derived/desktop_engagement_agg_v1/query.sql

```diff
@@ -4,6 +4,7 @@ SELECT
   distribution_id,
   locale,
   app_version,
+  isp_name,
```

Comment:
ISP will be client level only. There are too many distinct values and we don't want to make the aggregate tables unnecessarily large and slower to query.

Reply:
Got it, will take it out of the agg table, thanks!